styles/custom/variable.scss

```diff
@@ -1,4 +1,5 @@
 $padding: 20px;
+$paddingVW: 2vw;
```

Comment:
I decided to use vw at the time to solve the issue of when the width of the window is small the playlist and volume components begin to overlap each other.
I initially thought using vw might solve this (as the unit is based on the viewport width). However I did not get this attempt to work and forgot to revert the code.
I will try and solve this and create another issue/branch at some point. 
Anyway thanks for your feedback again.